src/index.css

```
@@ -1,18 +1,109 @@
+.pf-v6-c-page__main {
+  display: flex;
+  flex-direction: column;
+}
+
+.pf-v6-c-page__main > .pf-v6-c-page__main-section:last-child {
+  flex: 1;
+}
+
+.pf-v6-c-page > .pf-v6-c-masthead {
+  --pf-v6-c-masthead--BackgroundColor: var(--pf-t--color--gray--95);
+  display: flex;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)